### DISCUSSION TITLE
JSON Schema For App Manifest

### DISCUSSION THREAD

**Discussion by ghost** [ScoopInstaller/Scoop#5096] [Help and Questions] [answered]
I could not find any issues about an official _JSON Schema_ for app manifest (its code completion is helpful) so I preferred to discuss it here rather than as an open issue.
is there any schema for the app manifest?

**Comment by rashil2000** [comment, marked as answer]
https://github.com/ScoopInstaller/Scoop/blob/master/schema.json
